torch/_torch_docs.py

```diff
@@ -3274,10 +3274,11 @@ def merge_dicts(*dicts):
            r"""
 fmod(input, other, *, out=None) -> Tensor
 
-Computes the element-wise remainder of division.
+Computes the element-wise remainder of division with the remainder having the same
+sign as the dividend :attr:`input`.
 
-The dividend and divisor may contain both for integer and floating point
-numbers. The remainder has the same sign as the dividend :attr:`input`.
+.. math::
+    \text{{out}}_i = \text{{input}}_i - trunc(\frac{\text{{input}}_i}{\text{{other}}_}) * \text{{other}}_i
```

Comment:
Probably split the string here.

Similar to
https://github.com/pytorch/pytorch/blob/34d66183865c0ab189ac6962ef4277a1afd4bd2c/torch/_torch_docs.py#L192-L195

Reply:
Trying this fix, @kshitij12345 - will update if it works! Thank you.